config/initializers/env.rb

```diff
@@ -14,7 +14,11 @@
                                 else
                                   ''
                                 end
-  europeana[:entities_api_key] = ENV['EUROPEANA_ENTITIES_API_KEY'] || Europeana::API.key
+
+  europeana[:entities] = OpenStruct.new(
+    api_key: ENV['EUROPEANA_ENTITIES_API_KEY'] || Europeana::API.key,
+    api_url: ENV['EUROPEANA_ENTITIES_API_URL'] || Europeana::API.url
```

Comment:
This is causing problems for some of the specs that with the setup I have/had my .env files in.
I was setting `EUROPEANA_API_URL` to the acceptance api in .env and not setting the entities API url at all.
I guess, when running rspec, `Europeana::API.url` hasn't been set to the ENV value at the point when this initializer is ran, because the tests stubbed all the accpetance requests but when executing fixtures it was making requests to the default API url from the API ruby gem.

it's more robust if we do this:
```api_url: ENV['EUROPEANA_ENTITIES_API_URL'] || ENV['EUROPEANA_API_URL'] || Europeana::API.url```